services/thermal_mgr/thermal_mgr.c

```diff
@@ -43,21 +46,56 @@ void initThermalSystemManager(lm75bd_config_t *config) {
 
 error_code_t thermalMgrSendEvent(thermal_mgr_event_t *event) {
   /* Send an event to the thermal manager queue */
-
+  BaseType_t checkQueueSend;
+  if (event == NULL) {
+    return ERR_CODE_INVALID_ARG;
+  }
+  checkQueueSend = xQueueSend(thermalMgrQueueHandle, (void *) event, (TickType_t) 10);
+  if (checkQueueSend == pdFAIL) {
```

Comment:
this can be combined to just but `if(xQueueSend(thermalMgrQueueHandle, (void *) event, (TickType_t) 10) == pdFAIL)`